### DISCUSSION TITLE
What is the most performant way to show Preview of react component?

### DISCUSSION THREAD

**Discussion by nirtamir2** [codesandbox/sandpack#735] [Q&A] [answered]
I tried the vite-react template and the create-react-app example - and it seems the `create-react-app` is more performant compared to the `node` environment running vite
Am I missing something?

I added my examples here for reference:

[create-react-app-template-example
](https://codesandbox.io/s/sandpack-2-create-react-app-vwe5tm)

[vite-react-template-example
](https://codesandbox.io/s/sandpack-2-vite-6etgnd?file=/src/App.tsx)

What is the best way (in terms of performance) to show a preview for react component?

**Comment by danilowoz** [comment, marked as answer]
Hey! Definitely, I would go for the `create-react-app` template if you don't need server capabilities or run Vite on your examples. Plus, if you're interested in performance, you should give a try on the experimental bundler [here](https://sandpack.codesandbox.io/docs/advanced-usage/bundlers)

**Reply by nirtamir2** [reply]
Thanks!
